src/custom/state/claim/hooks/index.ts

```diff
@@ -849,3 +849,36 @@ function _getPrice({ token, amount }: { amount: string; token: Token | GpEther }
     quoteAmount: CurrencyAmount.fromRawAmount(token, amount),
   }).invert()
 }
+
+export function useTotalVCowAmount() {
+  const { chainId } = useActiveWeb3React()
+  const { selected, activeClaimAccount, investFlowData } = useClaimState()
+  const claims = useUserEnhancedClaimData(activeClaimAccount)
+
+  if (!chainId) {
+    return undefined
+  }
+
+  const zeroVCow = CurrencyAmount.fromRawAmount(V_COW[chainId], '0')
+
+  return claims.reduce<typeof zeroVCow>((totalTokens, claim) => {
+    const { price, currencyAmount } = claim
+    // if claim is included in selected invest option
+    if (selected.includes(claim.index)) {
```

Comment:
What about free claims?

Reply:
Based on text and place, I assume this is only to show investment claims sum `vCOW to receive based on selected investment(s):`